backend/tests/util.py

```diff
@@ -26,11 +27,14 @@
 import time
 import werkzeug.serving
 
+from .lorapatch import LORA_PATCHED
```

Comment:
:-1:

Reply:
This will have a short life. Pretty soon there will be integration data in all objects in lora. Using copious amounts of hours to fix something that will be gone in a split second seems a little over the top. But if You can get it to work without this, be my guest.

Reply:
I believe we'll eventually want this going forward, so I'll take a look!